eZ/Publish/SPI/Search/Handler.php

```diff
@@ -106,4 +106,23 @@ public function indexLocation(Location $location);
      * @param mixed $contentId
      */
     public function deleteLocation($locationId, $contentId);
+
+    /**
+     * Indexes several content objects.
+     *
+     * @param \eZ\Publish\SPI\Persistence\Content[] $contentObjects
+     */
+    public function bulkIndexContent(array $contentObjects);
```

Comment:
We need to figure out how to deal with warnings in a good way, it is clear from https://github.com/ezsystems/ezplatform-solr-search-engine/pull/47 that this api addtion is not good for that.

Possible solution:
- Instead expose on `SPI\Search\Handler` interface, pseudo code:
 - `use eZ\Publish\SPI\Search\Document;` 
 - `function mapDocumentByContent( Content $content ) : Document`
 - `function bulkIndexDocuments( Document[] $documents ) : void`
- Hence moving the part where you might get exception to indexer command, to log warnings.
 - We should also define exception `mapDocumentByContent` will throw on db inconsistency issues, this one can contain the relevant warning message for what went wrong, in addition to nesting the given `NotFound` exceptions etc.